src/app/pages/account/users/user-form/user-form.component.ts

```diff
@@ -362,6 +362,11 @@ export class UserFormComponent {
       if (!entityForm.isNew) {
         entityForm.setDisabled('uid', true);
         entityForm.formGroup.controls['username'].setValue(res[0].username);
+        // Be sure namesInUse is loaded, edit it, set username again to force validation
```

Comment:
this isn't the shark icon :)

Reply:
Oh, did it bring over changes when I change the branch? Oopsies

Reply:
How do I get rid of them?

Reply:
How do I get rid of them @erinix ?